subsys/net/lib/coap/coap_server.c

```diff
@@ -433,6 +434,9 @@ int coap_service_start(const struct coap_service *service)
 
 	coap_server_update_services();
 
+	net_mgmt_event_notify_with_info(NET_EVENT_COAP_SERVICE_STARTED, NULL, service,
+					sizeof(struct coap_service));
```

Comment:
Is it possible to know if the `sock_fd` is bound to a specific interface to be passed here?